src/axom/multimat/examples/calculate_gpu.cpp

```diff
@@ -117,7 +117,7 @@ void avgDensityCompactFlat(mmat::MultiMat& mm)
 {
   axom::fmt::print("Running average density compact - flat\n");
   int ncells = mm.getNumberOfCells();
-  int nmats = mm.getNumberOfMaterials();
+  // int nmats = mm.getNumberOfMaterials();
```

Comment:
@publixsubfan -- The `nmats` variables caused warnings b/c they weren't used. I wasn't sure if we wanted to keep them, e.g. for the future and/or for reference, so I just commented them out.

Reply:
matter of style: I usually '#if 0' things like this with a comment, so that it's easier to search for, during code cleanup for example.

Reply:
Thanks. I'm leaning towards just removing these, unless there's a good reason to keep them.

Reply:
Update: I removed these commented out lines. I'd be happy to add them back if desired.